doc/building.rst

```diff
@@ -209,14 +209,19 @@ shell), and run::
    $ nmake check
    $ nmake install
 
-Botan supports the nmake replacement `Jom <https://wiki.qt.io/Jom>`_
-which enables you to run multiple build jobs in parallel.
-
 For MinGW, use::
 
    $ python3 configure.py --cc=gcc --os=mingw
    $ make
 
+Botan supports the `ninja <https://ninja-build.org>`_  build system
```

Comment:
This should be a bit more specific to avoid confusion: "Starting in Botan 3.2, there is also support for ..."